security.rst

```diff
@@ -2556,7 +2556,7 @@ If you're having problems authenticating, it could be that you *are* authenticat
 successfully, but you immediately lose authentication after the first redirect.
 
 In that case, review the serialization logic (e.g. the ``__serialize()`` or
-``serialize()`` methods) on you user class (if you have any) to make sure
+``serialize()`` methods) in the user class (if you have any) to make sure
```

Comment:
On your ?

Reply:
Merged in b7392ebe91df3a3d46f4aceb0160541b57efee0c